config/env/development.js

```diff
@@ -12,6 +12,8 @@
 
 module.exports = {
 
+  appUrl: 'http://localhost:1337',
```

Comment:
development.js` has the port `1337` while production does not. Does this make sense?